### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by joh-ann** [joh-ann/overlook#9]
- [x] You must be able to tab through your app and use it without a mouse
- [x] Your app must still be usable when tested with a colorblind extension
- [x] You must score as close to 100% as possible with the “Lighthouse Accessibility Audit”. Be prepared to explain any accessibility audits your application is failing.
- [x] Your HTML must be written semantically and should use ARIA tags (ONLY if needed / appropriate)
